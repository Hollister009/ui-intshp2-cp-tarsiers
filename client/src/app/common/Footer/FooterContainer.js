import { connect } from 'react-redux';
import Footer from './Footer';

const mapStateToProps = state => ({
  headerFooterVisibility: state.headerFooterVisibility
});

export default connect(mapStateToProps)(Footer);
