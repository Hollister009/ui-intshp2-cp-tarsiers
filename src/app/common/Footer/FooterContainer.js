import { connect } from 'react-redux';
import Footer from './Footer';

const mapStateToProps = state => ({ ...state });

export default connect(mapStateToProps)(Footer);
