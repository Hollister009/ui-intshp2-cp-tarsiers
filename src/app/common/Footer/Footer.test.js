import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('<Footer />', () => {
  it('should match its snapshot', () => {
    const wrapper = shallow(<Footer />);

    expect(wrapper).toMatchSnapshot();
  });
});

describe('<Footer />', () => {
  it('should have headerFooterVisibility prop', () => {
    const wrapper = shallow(<Footer />);

    expect(wrapper.find({ prop: 'headerFooterVisibility' }));
  });
});
