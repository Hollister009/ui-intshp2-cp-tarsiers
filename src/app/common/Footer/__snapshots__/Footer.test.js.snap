// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should match its snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
